pkgs/applications/version-management/pijul/pijul.org.context

```diff
@@ -0,0 +1,92 @@
+
+Context:
```

Comment:
I think I was getting confused. I had a commit at one point that was pulling from `git` before they pushed that change to the `darcs` repo as well. I'm now getting "unknown tag 6ab9ba". Phew.

Sorry about the noise and thanks for the explanation (and the fix!)!